.github/workflows/operator-ci.yml

```diff
@@ -51,14 +51,21 @@ jobs:
           eval $(minikube -p minikube docker-env)
           docker build --build-arg KEYCLOAK_DIST=$(ls keycloak-*.tar.gz) . -t keycloak:${GITHUB_SHA}
 
+      - name: Build a custom pre-augmented Keycloak Docker image
+        run: |
+          echo -e "FROM keycloak:${GITHUB_SHA}\nRUN /opt/keycloak/bin/kc.sh build --db=postgres --health-enabled=true" > Dockerfile
+          eval $(minikube -p minikube docker-env)
+          docker build . -t custom-keycloak:${GITHUB_SHA}
```

Comment:
Since this is now a vital part of one test (it directly depends on it), wonder if we should make it more independent of GHA to be able to run it locally (and in the QE pipelines in the future) too. Maybe a shell script?

Reply:
What we can extract from here is the `Dockerfile`, I can commit it to the repo, but I was un-sure where to place it, wdyt about `operator/src/test/resources/Dockerfile`? any alternative?

Reply:
👍  for keeping it in ``. However, let's use a different name for the Dockerfile (maybe ``) as we might need more test images in the future.

Plus, I'd really create a shell script in the root of the operator module to make building it (with proper tags etc.) as easy and straightforward as possible.

Reply:
slightly improved the script and added a little documentation in the readme